acceptance-tests/src/test/java/tech/pegasys/signers/secp256k1/tests/multikey/HashicorpHelpers.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.signers.secp256k1.tests.multikey;
+
+import static java.util.Collections.singletonMap;
+
+import tech.pegasys.signers.hashicorp.dsl.HashicorpNode;
+import tech.pegasys.signers.secp256k1.HashicorpSigningParams;
+
+import com.github.dockerjava.api.DockerClient;
+
+public class HashicorpHelpers {
+
+  public static final String secretPath = "acceptanceTestSecretPath";
+  public static final String secretName = "value"; // this is the required default usedin EthSigner
```

Comment:
nit: bad comment - does "value" still need to be used? If so - why?

Reply:
No it doesn't have to be value at all. Signers doesn't care. I'll remove the comment